service_storagefacility/src/main/scala/no/uio/musit/microservice/storagefacility/resource/StorageNodeResource.scala

```diff
@@ -298,5 +298,8 @@ final class StorageNodeResource @Inject() (
     }
   }
 
+  def CurrentObjectLocation(mid: Int, oid: Long) = Action.async { implicit request =>
```

Comment:
not part of this issue ;-)

(and BTW, it should be `currentObjectLocation` with lowercase c)